src/noframe.js

```diff
@@ -12,22 +12,24 @@ export default function noframe(target, container) {
   if (!('length' in frames)) frames = [frames];
   for (let i = 0; i < frames.length; i += 1) {
     const frame = frames[i];
-    let parent = frame.parentElement;
     const h = frame.offsetHeight;
     const w = frame.offsetWidth;
     const styles = frame.style;
     let maxW = `${w}px`;
-    if (typeof container !== 'undefined') {
```

Comment:
What about something like? 
```javascript
const isContainerElement = typeof container !== 'undefined' && document.querySelector(container).length > -1)
const parent = isContainerElement ? document.querySelector(container) : frame.parentElement
if (isContainerElement) {...
```
I have not checked the snippet about but something like that. 🤓
...The reason why I prefer path is because it **is** "container" that is optional; not the "parent".

Reply:
Hey @yowainwright 

The Document method `querySelector()` returns the first Element within the document that matches the specified selector. If no matches are found, `null` is returned.

So `document.querySelector(container).length` will be undefined.

Also, why should we,

#1 -  Increase one more var `isContainerElement `
#2 - Call document.querySelector(container) twice.

I am submitting minimal changes, hope that would be fine.

Thanks,
-Mohan